packages/core/src/StellarHandler.js

```diff
@@ -40,6 +47,39 @@ though it processes the ${serviceInbox} queue`);
     this._startProcessing(url);
   }
 
+  _handleLoader(url, method, value) {
+    if (isArray(value)) {
```

Comment:
add test for the this function where value is an array (1, 2 & 3 elements), function and undefined